partner_contact_nutrition_diet/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Nutrition Diet",
+    "summary": "Set the nutrition diet of your contacts",
+    "version": "9.0.1.0.0",
+    "category": "Uncategorized",
```

Comment:
What do you think about putting these under `Health`? Seems like we have enough modules to warrant a category